Formula/openssh.rb

```diff
@@ -34,6 +34,9 @@ class Openssh < Formula
   def install
     ENV.append "CPPFLAGS", "-D__APPLE_SANDBOX_NAMED_EXTERNAL__"
 
+    # Ensure sandbox profile prefix is correct.
+    inreplace "sandbox-darwin.c", "@PREFIX@/share/openssh", etc/"ssh"
```

Comment:
To make the situation more fun, the two patches we apply to the `openssh` source code come from Apple, so _Apple is patching an upstream project's Darwin sandbox support_ 🤔 😓. I've given up trying to understand and moved onto trying to make it work, I found that less painful, heh.